### ISSUE TITLE
R1Q5 elliptical distortion

### ISSUE THREAD

**Issue by dagewa** [dagewa/DIALS_for_ED_paper#17]
> In a TEM the primary source of elliptical distortion is the projector lens. When SAED or nanodiffraction with a sufficiently large beam size is used, the settings of the projector lens for a given camera length are constant, and the distortion can be calibrated for. Did you see this? The distortion can also originate from a corrector – does the TEM used have a corrector? The information about the instrument used is missing.

**Comment by dagewa** [comment]
For the lysozyme data we used a Titan Krios with a Cs corrector. This corrector could have contributed to the elliptical distortion. The distortion was constant over various data sets.

**Comment by dagewa** [comment]
todo: check all information is covered in @cmaxtb's previous paper in the same journal. If so, highlight that point in the reply. No changes required to manuscript

**Comment by cmaxtb** [comment]
yes this should be already covered in the paper

**Comment by dagewa** [comment]
Reply:
The information about the instrument used is contained in the earlier paper Clabbers et al. 2017 (It was a Titan Krios with a Cs corrector that could have contributed to the elliptical distortion). The procedure used to calibrate the distortion is also described in that paper. As described in Section 2.4, we took the results of that calibration in order to create correction tables in a format specific for DIALS. A note has been added in this paper to indicate that the distortion was constant across 6 of the example datasets.
